docs/configuration/http_conn_man/http_conn_man.rst

```diff
@@ -97,6 +100,44 @@ http_codec_options
   <config_cluster_manager_cluster_http_codec_options>` option. See the comment there about
   disabling HTTP/2 header compression.
 
+.. _config_http_conn_man_http2_settings:
+
+http2_settings
+  *(optional, object)* Additional HTTP/2 settings that are passed directly to the HTTP/2 codec.
+  Currently supported settings are:
+
+  hpack_table_size
+    *(optional, integer)* `Maximum Table Size`_ (in octets) that the encoder is permitted to use for
```

Comment:
for consistency, can you change the links here to `link-name <link>`_. And delete the links at the end of this section.